src/gpodder/model.py

```diff
@@ -145,6 +145,14 @@ def from_podcastparser_entry(cls, entry, channel):
         episode.title = entry['title']
         episode.link = entry['link']
         episode.description = entry['description']
+        if entry.get('description_html'):
+            episode.description_html = entry['description_html']
+        # XXX: That's not a very well-informed heuristic to check
+        # if the description already contains HTML. Better ideas?
+        elif '<' in entry['description']:
+            episode.description_html = entry['description']
+        else:
+            episode.description_html = entry['description'].replace('\n', '<br>')
```

Comment:
Yes. Some podcasts provide an short (1 paragraph) description and a longer html_description with complete show notes and links. It seems bad to ignore their hand-crafted description and put a substring of the html_description when we can keep both.